web/src/main/java/org/openmrs/layout/web/address/AddressSupport.java

```diff
@@ -29,6 +29,8 @@
 @Deprecated
 public class AddressSupport extends LayoutSupport<AddressTemplate> implements GlobalPropertyListener {
 	
```

Comment:
I still don't understand why these changes are necessary. We expect the value from the DB to match the correct class name, if not then we need to fix the upgrade scripts to do so